src/pages/postgraphile/usage-cli.md

```diff
@@ -16,128 +16,147 @@ npm install -g postgraphile
 
 This will make the `postgraphile` command available. You can then run:
 
-```
+```bash
 postgraphile -c postgres://localhost/mydb -s public -a -j
 ```
 
 where `-c` is the connection string (defaults to postgres://localhost/), `-s` is the schema name (defaults to "public"), `-a` enables Relay support and `-j` enables dynamic JSON.
 
+PostGraphile also reads options from a `.postgraphilerc.js` file from the current working directory.
+
 ### CLI options
 
 There are more CLI options available to customise the GraphQL server:
 
 <!-- CLI_DOCBLOCK_BEGIN -->
 
-* `-h`, `--help`  
+* `-h`, `--help`
   output usage information
```

Comment:
Please restore these double spaces; they cause markdown to force a line break, I found this was the best way of formatting this page.